File: Backend/Data/Tables/Role.cs
using Microsoft.EntityFrameworkCore;
using System;
using System.Collections.Generic;

namespace Data.Tables
{
    public partial class Role
    {
        public Role()
        {
            Accounts = new HashSet<Account>();
        }

        public int Id { get; set; }
        public string? Name { get; set; }

        public virtual ICollection<Account> Accounts { get; set; }

        public static void Seed(ModelBuilder modelBuilder)
        {
            modelBuilder.Entity<Role>().HasData(
                new Role
                {
                    Id =1,
                    Name = "Admin"
                },
                new Role
                {
                    Id = 2,
                    Name = "Manager"
                },
                new Role
                {
                    Id = 3,
                    Name = "Teacher"
                },
                new Role
                {
                    Id = 4,
                    Name = "Student"
                }
                );
        }
    }
}

File: Frontend/ecv1/src/Constant/Role.ts
export const Role :any = (id: any) => {
    let name: string = '';
    switch (id) {
      case 1:
        name = 'Admin';
        break;
      case 2:
        name = 'Manager';
        break;
      case 3:
        name = 'Teacher';
        break;
      case 4:
          name = 'Student';
        break;
    }
    return name;
  };
